code/__DEFINES/flags.dm

```diff
@@ -51,6 +51,8 @@ GLOBAL_LIST_INIT(bitflags, list(1, 2, 4, 8, 16, 32, 64, 128, 256, 512, 1024, 204
 #define NOJAUNT_1 (1<<0)
 /// If a turf can be made dirty at roundstart. This is also used in areas.
 #define CAN_BE_DIRTY_1 (1<<1)
+/// Should we use the initial icon for display? Mostly used by overlay only objects
+#define HTML_USE_INITAL_ICON_1 (1<<20)
```

Comment:
you don't seem to have added this to anything

Reply:
also why is this in turf flags

Reply:
pasted from TG to make the code compile

Reply:
move this to be where it should
and add it to places where it is on tg

Reply:
leaving this comment as a reminder to myself to actually do this

Reply:
ok done